frontend/packages/process-editor/src/components/Canvas/BPMNEditor/BPMNEditor.test.tsx

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import { render, screen } from '@testing-library/react';
+import { BPMNEditor } from './BPMNEditor';
+import { BpmnApiContextProvider } from '../../../contexts/BpmnApiContext';
+import { BpmnConfigPanelFormContextProvider } from '../../../contexts/BpmnConfigPanelContext';
+import { textMock } from '../../../../../../testing/mocks/i18nMock';
+
+jest.mock('../../../hooks/useBpmnEditor', () => ({
+  useBpmnEditor: jest.fn().mockReturnValue({
+    canvasRef: { current: document.createElement('div') },
+  }),
+}));
+
+describe('BPMNEditor', () => {
+  it('render spinner when pendingApiOperations is true', () => {
+    render(
```

Comment:
Could this render method be refactored out in to its own function, ```renderBPMNEditor```, to avoid duplicating the 5 lines? By doing this we make it easier to build upon this test if we add more code later on 😄